src/main/resources/assets/tconstruct/book/encyclopedia/zh_cn/tools/ancient/tconstruct_swasher.json

```
@@ -0,0 +1,14 @@
+{
+  "tool": "tconstruct:swasher",
+  "text": [
+    { "text": "喷流刃是近战武器与手持流体枪的结合体。通常可以在埋藏的宝藏与沉船中获得；也可在溺尸手中，或是钓鱼获得。" }
```

```suggestion
    { "text": "喷流刃是近战武器与手持流体枪的结合体。通常可以在埋藏的宝藏与沉船中获得；也可从溺尸手中，或是钓鱼获得。" }
```
Polishing, recommended but not required.